internal/provider/resource_ably_rule_http_test.go

```diff
@@ -65,13 +67,15 @@ func TestAccAblyRuleHTTP(t *testing.T) {
 					"ably_api_key.api_key_1.id",
 					"https://example1.com/webhooks",
 					"msgpack",
+					"",
```

Comment:
I haven't worked on this provider before, but just looking at the surrounding code, it seems like this should be a true/false value, rather than a string which is either `enveloped = true` or an empty string?

Reply:
https://github.com/ably/terraform-provider-ably/pull/167/commits/147d471ad0e97b8b73aac9f5b0111943808ffea9

Reply:
👍